docs/scos/user/intro-to-spryker/whats-new/ie11-end-of-life.md

```diff
@@ -0,0 +1,32 @@
+---
+title: IE11 end of life
+description: Microsoft recently announced that the Internet Explorer 11 desktop application would be retired on June 15, 2022, for specific versions of Windows 10. Legacy Internet Explorer-based websites and applications will continue to work with Microsoft Edge’s built-in Internet Explorer mode.
+last_updated: June 8, 2022
+template: concept-topic-template
+originalLink: https://documentation.spryker.com/2021080/docs/ie11-end-of-life
+originalArticleId: I DO NOT KNOW WHAT TO PUT HERE
```

Comment:
@lenadoc What do we put inside the `originalArticleId` property?

Reply:
@jeremyfourna nothing for new docs. This property was used to set up redirects for atricles from the old docs portal to the current one.